docs/asset-source/test-telemetry.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+participant "Device App" as D
+participant "Service App" as S
+
+title **Telemetry Testing**
+
+D -> S: SendTelemetry(\n\tcmd="serviceAckRequest", \n\tserviceInstanceId=<instanceId>, \n\tserviceAckId=<ackId_1>)

```

Comment:
q - is all of this in the payload `(cmd="serviceAckRequest", serviceInstanceId=<instanceId>, serviceAckId=<ackId_1>)`? Is it a json?

Reply:
It is JSON in the payload.  I think I'll update this to be explicit.